lib/kaiser/cli.rb

```diff
@@ -582,8 +582,9 @@ def ensure_setup
           --name #{Config.config[:shared_names][:dns]}
           --network #{Config.config[:networkname]}
           --privileged
-          -v /var/run/docker.sock:/docker.sock:ro
-          phensley/docker-dns
+          -e DOCKER_HOST='unix:///var/run/docker.sock'
+          -v /var/run/docker.sock:/var/run/docker.sock:ro
+          davidsiaw/dockerdns
```

Comment:
Should we fork this into degica account instead?

Reply:
we can. i can do that in another pr. i actually have a nice build system and everything on that now